controller/lib/hal/startup_stm32l452xx.S

```diff
@@ -0,0 +1,103 @@
+/**
+  ******************************************************************************
+  * @file      startup_stm32l452xx.s
+  * @author    MCD Application Team

```

Comment:
This seems to be copied from somewhere. Could you indicate from where, and whether you made any changes?

Reply:
I copied it from the platformio files.  I've added a comment to that effect.